lib/travis/build/appliances/deprecate_xcode_64.rb

```diff
@@ -0,0 +1,16 @@
+require 'travis/build/appliances/base'
+
+module Travis
+  module Build
+    module Appliances
+      class DeprecateXcode64 < Base
+        def apply
+          if config[:osx_image] == 'xcode6.4'
+            sh.echo "Running builds with Xcode 6.4 in Travis CI is deprecated and will be removed in January 2019.", ansi: :yellow
```

Comment:
could you add a new line above the first line, and below the last line, to help it stand out more?

Reply:
Definitely!